src/booleans.js

```diff
@@ -1,59 +1,152 @@
 export const negate = (a) => {
-  // your code here
+    if (a == true) {
+    return false;
+    } else {
+    return true;
+    }
+    // returns the opposite of the passed boolean value
+    // expect(negate(true)).toBe(false)
 };
 
 export const both = (a, b) => {
-  // your code here
+    if (a && b == true) {
```

Comment:
Operator precedence means that `==` gets evaluated here before `&&`, hence you're only checking `b == true`. Again this could be done as `if (a && b)`